### ISSUE TITLE
GarminCustomMaps as published not acting as if a valid zip

### ISSUE THREAD

**Issue by kcrossen** [NINAnor/GarminCustomMaps#2]
As published here: https://plugins.qgis.org/plugins/GarminCustomMap/, click "Download latest", the resulting file downloaded to OS-X 10.11.6 fails to unpack w/ "The Unarchiver", which claims to be looking for other parts of archive. Some unzip tools will open GarminCustomMap-1.2.1.zip, but apparently not the one used internally by QGIS. I even changed the OS default to one that worked, no joy. The result is that I've not yet figured any way to install the plugin via the "official" method.

**Comment by ninsbl** [comment]
Hi @kcrossen, thanks for the bug report!
Unfortunately, I have no Mac available to test myself, so I would depend on your help to fix this...
Could you provide a full error message?

**Comment by kcrossen** [comment]
Good news, I think or hope. Yours was not the only victim, actually, and all have loaded today. They all had the behavior of the downloaded zip (manual download button) not being openable w/ the standard OS-X decompression utility, but were openable w/ **some** other decompression applications (but not all, I tried quite a few). I was suspecting a damaged zip file, the least complicated option. I also tried installing some other Plugins, e.g. OpenLayers, and they installed successfully. The other fails yesterday were NumericalDigitize and Numerical Vertex Edit, which both failed in exactly the same way and were tested with the same results as yours. Very sorry to have bother you w/ this, my new suspicion is that the Plugin server was wedged somehow.
